### PR TITLE
Adding how to expose elasticsearch service with a route to README.md.

### PR DESCRIPTION
**Pull request by nhosoi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,6 +141,18 @@ Decide how many nodes you want to run.
...
+
+Obtain the CA cert from Elasticsearch.
+```
+oc extract secret/elasticsearch --to=. --keys=admin-ca
+```
+Create a Secured Route Using Re-encrypt Termination.
+(See also https://docs.openshift.com/container-platform/3.11/architecture/networking/routes.html#secured-routes)
+
+In the Re-encrypt termination example, put the CA cert from Elasticsearch to spec.tls.destinationCACertificate.  
+The rest spec.tls.key, spec.tls.certificate and spec.tls.caCertificate are no need to be set.
```
Comment (mburke5678) [inline README.md]: s/The rest spec.tls.key, spec.tls.certificate and spec.tls.caCertificate are no need to be set./You do not need to set the spec.tls.key, spec.tls.certificate and spec.tls.caCertificate parameters shown in the example.

File: README.md [hunk 2]
```diff
@@ -141,6 +141,18 @@ Decide how many nodes you want to run.
 
 TODO
 
+### Exposing elasticsearch service with a route
+
+Obtain the CA cert from Elasticsearch.
```
Comment (mburke5678) [inline README.md]: @nhosoi Is the CA cert installed automatically or does the user need to download the CA cert somehow?

File: README.md (hunk 1 above)
Comment (nhosoi) [reply README.md]: Thanks!  Updating it...

File: README.md (hunk 2 above)
Comment (nhosoi) [reply README.md]: @mburke5678, the CA cert is supposed to be installed when the elasticsearch and other containers were installed.  We just need to retrieve the existing one.

Please see Rich's comment.  We may want to change the file name "admin-ca" to make it clearer...  Do you have any recommendation?
https://github.com/openshift/cluster-logging-operator/pull/104
> I think this is the CA cert for the CA that issued the Elasticsearch server cert. admin-ca is slightly misleading - we are assuming that the same CA was used to issue the ES admin cert and the ES http(s) server cert.

**Comment by nhosoi** [comment]
Note: https://jira.coreos.com/browse/LOG-283

File: README.md [hunk 3]
```diff
@@ -141,6 +141,18 @@ Decide how many nodes you want to run.
...
+```
+oc extract secret/elasticsearch --to=. --keys=admin-ca
+```
+Create a Secured Route Using Re-encrypt Termination.
```
Comment (jcantrill) [inline README.md]: We should say something like:

Create a [re-encrypt route](https://docs.openshift.com/container-platform/3.11/architecture/networking/routes.html#secured-routes).

File: README.md [hunk 4]
```diff
@@ -141,6 +141,18 @@ Decide how many nodes you want to run.
...
+(See also https://docs.openshift.com/container-platform/3.11/architecture/networking/routes.html#secured-routes)
+
+In the Re-encrypt termination example, put the contents of the file admin-ca to spec.tls.destinationCACertificate.
```
Comment (jcantrill) [inline README.md]: ...example, use the contents of.... for spec.tls....

Comment (nhosoi) [reply README.md]: Done.  Thanks, @jcantrill !

**Comment by mburke5678** [comment]
Added to end user docs:
https://jira.coreos.com/browse/OSDOCS-309

**Pull request merged**